### PR TITLE
Bump JRE version, rebase to ubuntu

### LINKED ISSUE #25: [FEAT] Update to Java 17
### Is this a new feature request?

- [X] I have searched the existing issues

### Wanted change

I'm planning to require JDK 17 at some point but it's already supported. Could you please release a prerelease / beta image using JDK 17 to prepare for that?

### Reason for change

Ensure future compatibility.

### Proposed code change

_No response_

### PR DESCRIPTION
**Pull request by Roxedus**
[linuxserverurl]: https://linuxserver.io
[linuxserver.io][linuxserverurl]

------------------------------

 - [x] I have read the [contributing](https://github.com/linuxserver/docker-nzbhydra2/blob/master/.github/CONTRIBUTING.md) guideline and understand that I have made the correct modifications

------------------------------

## Source / References:

closes #25

### CODE REVIEW

**Comment by thespad** [comment]
What's the plan for armhf? Deprecation will presumably be forced when 17 becomes required, unless we switch back to Ubuntu?

**Comment by Roxedus** [comment]
reverting to ubuntu is fine by me, did we have a written down list of the issues people where having on both OSs?

**Comment by thespad** [comment]
Alpine was the TCP DNS issue. Ubuntu was the jammy seccomp issue, which should have lessened by now as more people have updated their docker install.

File: Dockerfile [hunk 1]
```diff
@@ -8,44 +10,46 @@ LABEL build_version="Linuxserver.io version:- ${VERSION} Build-date:- ${BUILD_DA
...
-  apk add -U --no-cache \
-    curl \
-    jq \
-    openjdk11-jre-headless \
-    python3 && \
   echo "**** install nzbhydra2 ****" && \
   if [ -z ${NZBHYDRA2_RELEASE+x} ]; then \
-    NZBHYDRA2_RELEASE=$(curl -sX GET "https://api.github.com/repos/theotherp/nzbhydra2/releases" \
-    | jq -r '.[0] | .tag_name'); \
+    NZBHYDRA2_RELEASE=$(curl -sX GET "https://api.github.com/repos/theotherp/nzbhydra2/releases/latest" \
```
Comment (thespad) [inline Dockerfile]: Won't this only get stable releases?

File: Dockerfile [hunk 2]
```diff
@@ -8,44 +10,46 @@ LABEL build_version="Linuxserver.io version:- ${VERSION} Build-date:- ${BUILD_DA
...
-    "https://github.com/theotherp/nzbhydra2/releases/download/v${NZBHYDRA2_VER}/nzbhydra2-${NZBHYDRA2_VER}-linux.zip" && \
+    /tmp/nzbhydra2.zip -L \
+      "https://github.com/theotherp/nzbhydra2/releases/download/v${NZBHYDRA2_VER}/nzbhydra2-${NZBHYDRA2_VER}-linux.zip" && \
   mkdir -p /app/nzbhydra2/bin && \
   unzip /tmp/nzbhydra2.zip -d /app/nzbhydra2/bin && \
-  rm -rf \
-    /app/nzbhydra2/bin/upstart/ \
-    /app/nzbhydra2/bin/rc.d/ \
-    /app/nzbhydra2/bin/sysv/ \
-    /app/nzbhydra2/bin/systemd/ && \
```
Comment (thespad) [inline Dockerfile:37]: This should be kept, I need to add it to master. Maybe move it to the cleanup section?

**Review by thespad — APPROVED** [review]

**Pull request merged**